pallas-network/src/miniprotocols/localstate/queries_v16/mod.rs

```diff
@@ -80,6 +80,9 @@ pub struct SystemStart {
     pub picoseconds_of_day: u64,
 }
 
+#[derive(Debug, Encode, Decode, PartialEq)]
+pub struct EpochNo(#[n(0)] pub u32);
+
```

Comment:
why do we need this wrapper?
I would prefer to use the integer directly, unless there's some benefit I'm not seeing.

Reply:
this is for test only purpose to send cbor results and assertions more clear.

e.g

```rust
// send results
let result = AnyCbor::from_encode(EpochNo(8));


// asserts
assert_eq!(epoch, Epoch(8));
```

instead of passing 

```rust
// send results
let result = AnyCbor::from_encode([8]);

// asserts
assert_eq!(epoch, [8];
```

Reply:
done @scarmuega